ee/vellum_ee/workflows/display/tests/workflow_serialization/test_basic_tool_calling_node_serialization.py

```diff
@@ -0,0 +1,285 @@
+from deepdiff import DeepDiff
+
+from vellum_ee.workflows.display.workflows.get_vellum_workflow_display_class import get_workflow_display
+
+from tests.workflows.basic_tool_calling_node.workflow import BasicToolCallingNodeWorkflow
+
+
+def test_serialize_workflow():
```

Comment:
I would expect this test to pass with no changes even if we remove the first file